com.chartboost.mediation/Runtime/AdFormats/Banner/Unity/ChartboostMediationUnityBannerAd.cs

```diff
@@ -50,8 +59,11 @@ private void Start()
             {
                 LockToFixedSize(sizeType);
             }
-            
-            ChartboostMediation.DidStart += ChartboostMediationOnDidStart;
+
+            if (autoLoadOnInit)
+            {
+                AutoLoadOnInit();
```

Comment:
Yeah, let's double-check if this is a feature request or if we want to make this a public API.